independent-publisher-connectors/SOS Inventory/apiProperties.json

```diff
@@ -0,0 +1,38 @@
+{
+	"properties": {
+		"connectionParameters": {
+			"token": {
+				"type": "oauthSetting",
+				"oAuthSettings": {
+					"identityProvider": "oauth2",
+					"clientId": "[[DUMMY]]",
+					"ClientSecret": "[[DUMMY]]",
+					"scopes": [],
+					"redirectUrl": "https://logic-apis-[parameters('location')].consent.azure-apim.net/redirect",
```

Comment:
@Harold-Anderson the redirectUrl property should have the following value: `"https://global.consent.azure-apim.net/redirect"`. Also please add this property and value `"redirectMode": "Global"` above the redirectUrl